definitions/variable/climate/magicc.yaml

```
@@ -0,0 +1,1376 @@
+- AR6 climate diagnostics|Atmospheric Concentrations|CH4|MAGICCv7.5.3|10.0th Percentile:
```

For consistency with the variable-notation guidelines.
```suggestion
- AR6 Climate Diagnostics|Atmospheric Concentrations|CH4|MAGICCv7.5.3|10.0th Percentile:
```

---

Is this PR still meant to be merged?

And if so, I suppose this capitalisation be changed directly in iiasa/climate-assessment?
I don't have any capacity for this right now (it's a small update, but updating the test-data is a bit of a pain, and just need to do a quick check to make sure it wouldn't break anything in notebooks or other scripts, and backward-compatibility).

---

For now, I wouldn't say it's super urgent. MAGICC variables are currently added after the variable validation so this PR is good to have for sure, but not essential right now.